pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -338,6 +341,7 @@ func (l *listUsersQuery) expandTTU(
 				Object:               &openfgav1.Object{Type: userObjectType, Id: userObjectID},
 				Relation:             computedRelation,
 				UserFilters:          req.GetUserFilters(),
+				ContextualTuples:     req.GetContextualTuples(),
```

Comment:
If we're going to add the tests for ABAC Conditions in the `_test.go` file at this time, then we should also do our diligence to wire up the `Context` here and everywhere else in this file.

Otherwise we should post pone the support for Conditions and the tests for them until a subsequent PR.

Reply:
We have a ticket specific to conditions work so would prefer that be done then. The tests in this PR were a first pass to confirm that yes, conditions don't work. We can always add more tests when ^ work is done.